Hippo.Core/Domain/Tag.cs

```diff
@@ -0,0 +1,16 @@
+using System.ComponentModel.DataAnnotations;
+
+namespace Hippo.Core.Domain;
+
+public class Tag
+{
+    public int Id { get; set; }
```

Comment:
Should this have the [Key] attribute?

Reply:
Not really necessary, since it's covered by convention.